README.md

```diff
@@ -43,10 +43,22 @@ Testing Data
 - Raw science frame of a few stars
 - https://figshare.com/articles/dataset/IRIS_Liger_DRS_Test_Data/26492029?file=48191524
 
-#### raw_background_20240829.fits
+#### raw_background_20240829.fitsg
```

Comment:
typo

Reply:
Fixed in [4f5f9d7](https://github.com/oirlab/liger_iris_pipeline/pull/76/commits/4f5f9d775e4e6b986cc1b74a0c9456e165935c92).

Correct, I only updated metadata (namely DATAMODL & DATE keys). I do have a function to address #25 when the time comes, but let's do that + functionality to generate fake metadata in one go.